### PR TITLE
Performative_Inclusion_Munoz

### PR DESCRIPTION
**Pull request by codexguajil**

### CODE REVIEW

File: 1811/Performative_Inclusion_Munoz [hunk 1]
```diff
@@ -0,0 +1,23 @@
+##OUTLINE
+###Each presentation should include 5-8 minutes of polished content with slides.
+
+Link to your slide deck (google slides recommended): https://docs.google.com/presentation/d/16rHDXK5nedb-SWwGMSk3SsbZ-hhNSUnoScYMosuEZCI/edit#slide=id.p
```
Comment (emhickmann) [inline 1811/Performative_Inclusion_Munoz:4]: Make sure this is viewable with the link!

File: 1811/Performative_Inclusion_Munoz [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Name of your Gear Up Presentation: Performative Inclusion
+
+Introduction of yourself: Hello, my name is Edgar Munoz. I am in mod4 of this program and I am going to talk to you all about something called performative inclusion.
+
+Why did you select this topic? Why is it of interest? The reason I selected this topic is because I have had conversations with women and poc who work in the industry. We know that the industry is trying to change. Turing is ahead of the curve in that respect. And I want to talk about some of the pitfalls orgs can fall into when they think they are being inclusive/are trying to be inclusive.
```
Comment (emhickmann) [inline 1811/Performative_Inclusion_Munoz:10]: This is a great intro!

File: 1811/Performative_Inclusion_Munoz [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Detailed description of the topic/history of the topic: First exclusion and examples, then attempts at inclusion and their shortcomings, where we are now, what you can do next.
```
Comment (emhickmann) [inline 1811/Performative_Inclusion_Munoz:12]: Excellent outline and make sure to highlight what we can do next.  Often people need those next steps.

File: 1811/Performative_Inclusion_Munoz [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+How does this connect to mission of Gear Up? The goal of technology, the goal of Turing is to bring more perspectives to developing technology so that our civilization uses all of its smart people from all backgrounds to create progress. Maybe talk about how inclusion initiatives have helped me.
```
Comment (emhickmann) [inline 1811/Performative_Inclusion_Munoz:14]: You could make it a specific connection to yourself.  That might even be good to put in your intro as to why this matters to you.

File: 1811/Performative_Inclusion_Munoz [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+How will this impact you as a developer? Will help me to continue to bridge the gap between where we are and where we should be in a nuanced, professional way.
```
Comment (emhickmann) [inline 1811/Performative_Inclusion_Munoz:16]: Could it also impact products?  Outcomes?  Access for all people to new ways of thinking, doing, interacting?

File: 1811/Performative_Inclusion_Munoz [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+Write 2-3 discussion questions you might want people to discuss after your presentation: 
+1. What does inclusion feel like to you? (Start by me talking about what exclusion feels like)
+2. What could an organization do better to help you continue to feel that feeling of being included
+3. Do you feel you are smarter + more creative in an inclusive space or in a not so inclusive space? Why?
```
Comment (emhickmann) [inline 1811/Performative_Inclusion_Munoz:21]: Great questions!!  Love these!

**Review by emhickmann — COMMENTED** [review]
@criteriamor This is going to be EXCELLENT!  Try to get content into your slides before next week so you can narrow it down and focus on your talking points.

**Comment by codexguajil** [comment]
Slides link: https://docs.google.com/presentation/d/16rHDXK5nedb-SWwGMSk3SsbZ-hhNSUnoScYMosuEZCI/edit?usp=sharing

**Pull request merged**